Solutions/Nasuni/Analytic Rules/RansomwareAttackDetected.yaml

```diff
@@ -0,0 +1,50 @@
+id: 6c8770fb-c854-403e-a64d-0293ba344d5f
+name: Ransomware Attack Detected
+description: 'Identifies ransomware attacks detected by the Ransomware Protection service running on a Nasuni Edge Appliance.'
+kind: Scheduled
+severity: High
+requiredDataConnectors:
+  - connectorID: NasuniEdgeAppliance
```

Comment:
Hello @mdriscoll-nasuni please correct the spelling of connectorid to ' connectorId ' only I should be capital in the word